app/models/media_object.rb

```diff
@@ -116,6 +116,9 @@ def validate_date(date_field)
   property :lending_period, predicate: ::RDF::Vocab::SCHEMA.eligibleDuration, multiple: false do |index|
     index.as :stored_sortable
   end
+  property :series, predicate: ::RDF::Vocab::SCHEMA.Series, multiple: true do |index|
```

Comment:
We may want to ask Julie if there is a more appropriate predicate, maybe through EBUCore or otherwise, but this one should be alright.